eav/admin.py

```diff
@@ -1,52 +1,49 @@
-'''Admin. This module contains classes used for admin integration.'''
+'''This module contains classes used for admin integration.'''
 
 from django.contrib import admin
-from django.contrib.admin.options import InlineModelAdmin
-from django.contrib.admin.options import ModelAdmin
+from django.contrib.admin.options import InlineModelAdmin, ModelAdmin
 from django.forms.models import BaseInlineFormSet
 from django.utils.safestring import mark_safe
 
 from .models import Attribute, EnumGroup, EnumValue, Value
 
 
 class BaseEntityAdmin(ModelAdmin):
-
-    def render_change_form(self, request, context, add=False, change=False, form_url='', obj=None):
-        """
-        Wrapper for ModelAdmin.render_change_form. Replaces standard static
-        AdminForm with an EAV-friendly one. The point is that our form generates
+    def render_change_form(self, request, context, add=False,
+                           change=False, form_url='', obj=None):
+        '''
```

Comment:
Aren't we supposed to change them the other way round, to PEP8 recommended """'s?

Reply:
Yup. When I started cleaning-up I thought to keep consistent with the majority of the code so I started changing to `'''`. Then I decided, actually no, let's keep consistent with the PEP. :smile: